### DISCUSSION TITLE
Bug with background image using arbitrary values

### DISCUSSION THREAD

**Discussion by someBrown** [tailwindlabs/tailwindcss#11738] [Help]
**What version of Tailwind CSS are you using?**

 "tailwindcss": "^3.3.3",

**What build tool (or framework if it abstracts the build tool) are you using?**
"postcss": "^8.4.27",
 "vite": "^4.3.2"

**What version of Node.js are you using?**

v16.20.0

**What browser are you using?**

Chrome

**What operating system are you using?**

macOS

**Reproduction URL**

**Describe your issue**
document location https://tailwindcss.com/docs/background-image#arbitrary-values

I refer to the documentation and use it like this in HelloWord.vue ,but the final resolved path is not correct.
The webpack project also has this problem

**Is it only possible to use absolute path?**

``` html
<template>
    <div class="bg-[url('./img/cloud1.png')]">
        <span>hello</span>
        <span>world</span>
    </div>
</template>

<style>

span{
    background: url('./img/cloud1.png') no-repeat;
}
</style>
```
Comparison of the URLs of the two background images requested by the browser
```
first 
http:localhost:5173/img/cloud1.png    404 Not Found

second
http:localhost:5173/src/components/img/cloud1.png 200
```

```
// directory structure
components(dir)
        --HelloWord.vue
        --img(dir)
             --cloud1.png

```

```js
//tailwind.config.js
export default{
    content: ['./src/**/*.vue'],
    theme: {},
    plugins: [],
  };
```

```js
//postcss.config
export default {
    plugins: {
      tailwindcss: {},
      autoprefixer: {},
    },
  }
```

``` js
//vite.config.js
/* eslint-disable import/no-extraneous-dependencies */
import { defineConfig } from 'vite'
import vue from '@vitejs/plugin-vue'

// https:vitejs.dev/config/
export default defineConfig({
  plugins: [vue()],
})
```

**Comment by AhmedBaset** [comment]
That is expected.

In the first example, the class class="bg-[url('./img/cloud1.png')]" in Tailwind CSS is transformed by the Tailwind CLI into a corresponding CSS class in the styles file, like this:

```css
.bg-\[url\(\'\.\/img\/cloud1\.png\'\)\] {
  background-image: url('./img/cloud1.png');
}
```
As a result, the image URL is relative to the `src/styles.css` file.

However, in the second example, the class is directly defined inside the `HelloWorld.vue` file:

```vue
<style>
  span{
     background: url('./img/cloud1.png') no-repeat;
   }
</style>
```

So the url is relative to the `HelloWorld.vue` file.
